Committing all files for Rules Publish from TST to PRD documentation

guzumba15 opened the pull request:
Committing all files for Rules Publish from TST to PRD documentation

guzumba15 committed:
Committing all files for RulesPublishTstToPrd documentation

Izzmo commented on README.md:
```diff
@@ -9,7 +9,7 @@ This information can be updated quite often, but is all peer reviewed. If you se
 * [Sprint Development](process/sprints.md)
 * [Feature Toggles](process/feature-toggles.md)
-
+* [Deployment](deployment/Rules-Publish-TST-To-PRD.md)
```
This link should go to a page that houses all documentation for deployments. Then, on that page, we link to your publish for rules document. This will allow us to expand it in the future. For now, this might be fine though.

Izzmo commented on deployment/Rules-Publish-TST-To-PRD.md:
```diff
@@ -0,0 +1,48 @@
+# Rules Publish from TST to PRD
+
+1.	Remote desktop into **TDCI01SQL**
```
Don't use tabs, just a single space should work.

Izzmo commented on deployment/Rules-Publish-TST-To-PRD.md:
```diff
@@ -0,0 +1,48 @@
+1.	Remote desktop into **TDCI01SQL**
+
+2.	Using SQL Server Management Studio connect to the **TDCI01SQL** database server
+
```
Can remove extra lines in between numbered items.

guzumba15 replied:
I'll remove some of these extra lines but the ones tied to an image I'm going to leave them. This way they're separated in a way its obvious which number the image goes to.

Izzmo commented on deployment/Rules-Publish-TST-To-PRD.md:
```diff
@@ -0,0 +1,48 @@
+
+4.	Open a command prompt and copy the file below into the window and hit enter
+    1.	C:\Infor\CopyDatabase - codeDEV to PRD.bat
+    ![Windows Batch File Copy Database](../images/WindowsBatchFileCopyDatabaseCodeDevToPrd.png)
```
These images are not working.

guzumba15 replied:
@Izzmo images work for me in the preview of Visual Studio Code? Where are you seeing its not working? It also looks like all the images were committed?

Izzmo commented on deployment/Rules-Publish-TST-To-PRD.md:
```diff
@@ -0,0 +1,48 @@
+
+![Restore CodeDev To TestModel](../images/RestoreCodeDevToTestModel.png)
+
+6.	Using SQL Server Management Studio connect to the **TDCI01SQLPROD** database server
```
I would make use of headings to say something like:

## Deploy to Test Rules Server

... numbered items

## Deploy to Production Rules Server

... numbered items

Izzmo reviewed (changes requested):
Pretty good, but images aren't showing up and formatting could be better.

guzumba15 committed:
PR changes adding a main deployments page.

guzumba15 commented:
@Izzmo Not sure where your comments went but I believe I've adjusted everything correctly. I added a main Deployments page which is called by the README.md and then that page calls the new documentation.
